tck/src/main/java/org/eclipse/microprofile/openapi/apps/airlines/resources/ReviewResource.java

```diff
@@ -161,10 +161,10 @@ public Response getReviewById(
     @Produces("application/json")
     public Response getReviewByUser(
             @Parameter(name = "user", description = "username of the user for the reviews", in = ParameterIn.PATH,
-                       content = @Content(examples = @ExampleObject(name = "example", value = "bsmith")), examples = {
+                       content = @Content(examples = {
                                @ExampleObject(name = "example1", value = "bsmith"),
                                @ExampleObject(name = "example2",
-                                              value = "pat@example.com")}) @PathParam("user") String user,
+                                              value = "pat@example.com")})) @PathParam("user") String user,
```

Comment:
This slightly expands the test as we're now testing for multiple examples within the `@Content`.

Reply:
Restore to single example with the parameter examples removed.